pkg/term/quiet.go

```diff
@@ -0,0 +1,42 @@
+package term
+
+import (
+	"github.com/acorn-io/runtime/pkg/install/progress"
+	"github.com/sirupsen/logrus"
+)
+
+type QuietBuilder struct {
+}
+
+func (b *QuietBuilder) New(msg string) progress.Progress {
+	return newQuietSpinner(msg)
+}
+
+type quietSpinner struct {
+	text string
+}
+
+func newQuietSpinner(text string) *quietSpinner {
+	return &quietSpinner{
+		text: text,
+	}
+}
+
+func (s *quietSpinner) Fail(err error) error {
+	if err != nil {
+		logrus.Errorf("Error encountered during '%v': %v", s.text, err)
+	}
+	return err
+}
+
+func (s *quietSpinner) Infof(format string, v ...any) {
+	// No-op, we're being quiet
+}
+
+func (s *quietSpinner) SuccessWithWarning(format string, v ...any) {
+	// No-op, we're being quiet
+}
+
+func (s *quietSpinner) Success() {
+	// No-op, we're being quiet
+}
```

Comment:
nit:

```suggestion
func (*quietSpinner) Infof(string, ...any) {
	// No-op, we're being quiet
}

func (*quietSpinner) SuccessWithWarning(string, ...any) {
	// No-op, we're being quiet
}

func (*quietSpinner) Success() {
	// No-op, we're being quiet
}
```

Reply:
took this suggestion, but locally and then force pushed